spec/mailers/user_mailer_spec.rb

```diff
@@ -65,6 +65,17 @@
         "[Reservations] #{@res.equipment_model.name} Request Approved")
     end
 
+    it 'sends approved request notifications for requests starting today' do
```

Comment:
Hmm, this is good but we're not actually checking the system response when the admin actually approves the request (e.g. `GET`s `/reservations/:id/approve`). That would probably require an integration test; if you feel comfortable with that feel free to try and write one, otherwise I'll add it to #1223.